### PR TITLE
uGMT online DQM muon copy validation

### PR DESCRIPTION
**Pull request by thomreis**
Online DQM histograms to validate the uGMT muon copies 1-5 against uGMT muons.
The additional copies are sent to uGT boards 1-5 and will be read out for validation events.
This checks that the collections contain the same muons.

Additionally, a histogram title is corrected.

### CODE REVIEW

**Comment by dmitrijus** [comment]
+1

File: DQM/L1TMonitor/python/L1TStage2uGMT_cfi.py [hunk 1]
```diff
@@ -117,3 +117,41 @@
     verbose = cms.untracked.bool(False),
 )
 
+# The five modules below compare the primary unpacked uGMT muon collection to goes to uGT board 0
```
Comment (davidlange6) [inline DQM/L1TMonitor/python/L1TStage2uGMT_cfi.py]: hi @thomreis @rekovic - this file needs to be renamed as a "cff" not a "cfi" (which should define one module). Please have a look in a near-term followup pr. thanks

**Comment by davidlange6** [comment]
+1

File: DQM/L1TMonitor/python/L1TStage2uGMT_cfi.py (hunk 1 above)
Comment (thomreis) [inline DQM/L1TMonitor/python/L1TStage2uGMT_cfi.py]: @davidlange6 so I should make one cfi file for every module and then group them in a cff?

**Pull request merged**